domain/schema/model/sql/0018-application.sql

```diff
@@ -24,16 +24,27 @@ CREATE UNIQUE INDEX idx_application_name
 ON application (name);
 
 CREATE TABLE cloud_service (
+    uuid TEXT NOT NULL PRIMARY KEY,
     application_uuid TEXT NOT NULL,
+    net_node_uuid TEXT NOT NULL,
     provider_id TEXT NOT NULL,
-    CONSTRAINT fk_cloud_application
+    CONSTRAINT fk_cloud_service_application
     FOREIGN KEY (application_uuid)
-    REFERENCES application (uuid)
+    REFERENCES application (uuid),
+    CONSTRAINT fk_cloud_service_net_node
+    FOREIGN KEY (net_node_uuid)
+    REFERENCES net_node (uuid)
 );
 
-CREATE UNIQUE INDEX idx_cloud_service_application
+CREATE UNIQUE INDEX idx_cloud_service_provider
+ON cloud_service (provider_id);
+
+CREATE INDEX idx_cloud_service_application
```

Comment:
In other index's in this PR we would take the form `application_uuid` and here we do `application`. Is it possible to get some consistency. I thought we were always going granular into the domain.

Reply:
This one is ok, the net node one below has the typo, thanks for finding it.